<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Student_Dialog</class>
 <widget class="QDialog" name="Student_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #73C2FB; /* Ice Blue*/
    font-family: 'Segoe UI', sans-serif;
}</string>
  </property>
  <widget class="QLineEdit" name="Welcome">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI,sans-serif</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
 {
    color: #FAFAD2;
    font-size: 24px;
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Welcome to Spartan Ride</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Spartan_Ride_App/SpartanLogo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: #FAFAD2; /* Gold */
    color: #0B1D3A;
    border: none;
    border-radius: 10px;
    padding: 10px;
    font-weight: bold;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Images/person.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="student_name">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
 {
    color: #FAFAD2;
    font-size: 12px;
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="Balamce">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>40</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
 {
    color: #FAFAD2;
    font-size: 12px;
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Balance</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>125</y>
     <width>701</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="Student_pushbuttons">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>701</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="Wallet">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #FAFAD2; /* Gold */
    color: #0B1D3A;
    border: none;
    border-radius: 10px;
    padding: 10px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #FAFAD2;
}</string>
    </property>
    <property name="text">
     <string>Wallet</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RideHistory">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #FAFAD2; /* Gold */
    color: #0B1D3A;
    border: none;
    border-radius: 10px;
    padding: 10px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #FAFAD2;
}</string>
    </property>
    <property name="text">
     <string>Ride History</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BookaRide">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #FAFAD2; /* Gold */
    color: #0B1D3A;
    border: none;
    border-radius: 10px;
    padding: 10px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #FAFAD2;
}</string>
    </property>
    <property name="text">
     <string>Book Ride</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Profile">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #FAFAD2; /* Gold */
    color: #0B1D3A;
    border: none;
    border-radius: 10px;
    padding: 10px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #FAFAD2;
}</string>
    </property>
    <property name="text">
     <string>Profile</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Feedback">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI,sans-serif</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #FAFAD2; /* Gold */
    color: #0B1D3A;
    border: none;
    border-radius: 10px;
    padding: 10px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #FAFAD2;
}</string>
   </property>
   <property name="text">
    <string>Feedback</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SignOutButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>590</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI,sans-serif</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #FAFAD2; /* Gold */
    color: #0B1D3A;
    border: none;
    border-radius: 10px;
    padding: 10px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #FAFAD2;
}</string>
   </property>
   <property name="text">
    <string>Sign Out</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>711</width>
     <height>261</height>
    </rect>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <property name="rowCount">
    <number>8</number>
   </property>
   <property name="columnCount">
    <number>6</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>Trip Id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Status</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>SJSU Id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Driver Id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Route</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Start Time</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="RideHistoryText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
 {
    color: #FAFAD2;
    font-size: 20px;
    font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Ride History</string>
   </property>
  </widget>
  <widget class="QPushButton" name="HomeButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>590</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI,sans-serif</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #FAFAD2; /* Gold */
    color: #0B1D3A;
    border: none;
    border-radius: 10px;
    padding: 10px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #FAFAD2;
}</string>
   </property>
   <property name="text">
    <string>Home </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
